repository/src/main/java/org/alfresco/repo/content/AbstractRoutingContentStore.java

```diff
@@ -420,23 +421,71 @@ public boolean delete(String contentUrl) throws ContentIOException
         return deleted;
     }
 
+    /**
+     * {@inheritDoc}
+     */
     @Override
     @Experimental
     public Map<String, String> getStorageProperties(String contentUrl) {
         ContentStore contentStore = selectReadStore(contentUrl);
 
         if (contentStore == null) {
-            if (logger.isTraceEnabled()) {
-                logger.trace("Storage properties not found for content URL: " + contentUrl);
-            }
+            logNoContentStore(contentUrl);
             return Collections.emptyMap();
         }
+        final String message = "Getting storage properties from store: ";
+        logExecution(contentUrl, contentStore, message);
+
+        return contentStore.getStorageProperties(contentUrl);
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    @Experimental
+    public boolean requestSendContentToArchive(String contentUrl)

```

Comment:
See earlier comment (re: leaving open the option to send params) ?

Reply:
See related comment/reply.

Reply:
Ditto ;-)